index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>J & M</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="content">
+      <div class="three_bg"></div>
+
+      <header>
+        <div class="nav">
+          <div class="nav-list">
+            <li>
+              <a href="#"><span>Home</span></a>
+            </li>
+            <li>
+              <a href="#"><span>About Us</span></a>
+            </li>
+            <li>
+              <a href="#"><span>Projects</span></a>
+            </li>
+            <li>
+              <a href="#"><span>Contact</span></a>
+            </li>
+          </div>
+        </div>
+      </header>
+      <div class="description">
+        <h1>Welcome to</h1>
+        <h2>
+          Jacob & Marwan Technical Solutions
+        </h2>
```

Comment:
[Optional} Please Add a button to the page that would navigate the user to another page

Reply:
- [ ] Nicey tried, but not so clear at all, hey @MarwanDev, I think the student will not understand why you are asking him to add a button and where? please be more precise and specific, you have told the student what he should do but it is required to explain `why` the student should apply your comment.